### PR TITLE
feat(wsl): pass verbose to topgrade-in-wsl

### LINKED ISSUE #521: `-v` should be passed to WSL when it is passed on host
## I want to suggest some general feature
Topgrade should pass `-v` / `--verbose` to the WSL step when it's passed to the main `topgrade` command.

### PR DESCRIPTION
**Pull request by ssmendon**
Closes #521

## Standards checklist:

- [x] The PR title is descriptive.
- [x] I have read `CONTRIBUTING.md`
- [x] The code compiles (`cargo build`)
- [x] The code passes rustfmt (`cargo fmt`)
- [x] The code passes clippy (`cargo clippy`)
- [x] The code passes tests (`cargo test`)
- [x] *Optional:* I have tested the code myself
 
## For new steps
- [ ] *Optional:* Topgrade skips this step where needed
- [ ] *Optional:* The `--dry-run` option works with this step

If you developed a feature or a bug fix for someone else and you do not have the
means to test it, please tag this person here.

@SteveLauC

---

I'd appreciate any feedback about this PR.

### CODE REVIEW

File: src/steps/os/windows.rs [hunk 1]
```diff
@@ -128,12 +128,40 @@ fn upgrade_wsl_distribution(wsl: &Path, dist: &str, ctx: &ExecutionContext) -> R
...
+    //  .arg(format!("exec {topgrade}"));
+    // ```
+    //
+    // creates a command string like:
+    //  `bash -c 'exec /bin/topgrade'`
+    // and using `command.args(...).arg("topgrade").arg("-v")`
+    // on that again creates
+    //  `bash -c 'exec /bin/topgrade' -v`
+    // which means `-v` isn't passed to `topgrade`.
+    // ```
```
Comment (SteveLauC) [inline src/steps/os/windows.rs:156]: 1. It seems that the command string created by the following code
    
    ```rust
    command
        .args(["bash", "-c"])
        .arg(format!("exec {topgrade}"));
    ```

    should be: `wsl bash -c 'exec /bin/topgrade'`

    This should also be applied to `command.args(...).arg("topgrade").arg("-v")`

2. The ``` in line 156 seems to be unmatched, and thus should be removed?

**Comment by SteveLauC** [comment]
Sorry for the late review, except for the comment nitpick, this PR LGTM! Thanks!

BTW, thanks for the comment, it helps code review a lot! :)

File: src/steps/os/windows.rs (hunk 1 above)
Comment (ssmendon) [reply src/steps/os/windows.rs:157]: Thanks for the proofreading. :)

I pushed a new commit, copying some output from my machine for the sample outputs. Let me know if this looks good: https://github.com/topgrade-rs/topgrade/pull/556/commits/d0d2ac9d9cf2479ea70422dd886cbeefca33d4bd

**Comment by SteveLauC** [comment]
LGTM! Thanks :)

**Comment by ssmendon** [comment]
I did the previous comment fixes in the GitHub UI. Since I noticed it failed the rustfmt check, I pulled it down and did everything for submitting a PR.

**Review by SteveLauC — APPROVED** [review]

**Pull request merged**